src/ray/rpc/grpc_server.h

```diff
@@ -86,6 +73,48 @@ class GrpcServer {
   std::unique_ptr<grpc::Server> server_;
 };
 
+/// Base class that represents an abstract gRPC service.
+///
+/// Subclass should implement `InitServerCallFactories` to decide
+/// which kinds of requests this server should accept.
+class GrpcService {
+ public:
+  /// Constructor.
+  ///
+  /// \param[in] name Name of this server, used for logging and debugging purpose.
```

Comment:
This docstring is for the GrpcServer instead of the GrpcService I think :)

Reply:
yes. Thanks for catching it. I also fixed a couple of other stale comments:)